### ISSUE TITLE
Source code publishing

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap-polska/sotm-eu-2024#7]
As I understand this repo contains HTML of website that was generated using webflow

Is source code (used to generate HTML) available somewhere? So it could be modified in case of changes being needed?

**Comment by ttomasz** [comment]
Webflow is WYSIWYG editor that allows exporting webpage code. Exported code is copied manually into www folder in this repo.

**Comment by starsep** [comment]
AFAIK Webflow is low-/no-code tool.
I haven't used it myself and I don't have access there.
I don't know whether there exists "source code". 
They definitely have their internal representation

If something is urgent I think it's ok to modify here but that needs to be also fixed in Webflow

**Comment by matkoniecz** [comment]
ouch

is it at least open source tool? Separate repo where internal state of webflow is committed could be also useful, so in case of some update it is not hanging on one person and/or requiring editing raw html.

Or maybe it is not too late to use Wordpress? So things can be fixed within reasonable time?

**Comment by ttomasz** [comment]
> ouch
> 
> is it at least open source tool? Separate repo where internal state of webflow is committed could be also useful, so in case of some update it is not hanging on one person and/or requiring editing raw html.

No, it's commercial software. It allows multiple editors in the paid plan. As far as vendor lock in we have the full html/css/js bundle exported from that we can use and do whatever we want with it (it's commited to this repo).

This was the easiest for our designer to provide style for the website so that's why it was used. I'm open to other arrangements but at the moment we really only have one designer so if other workflow will not work for them then that's going to be a problem.

> Or maybe it is not too late to use Wordpress? So things can be fixed within reasonable time?

I don't really care where and how the website is hosted. If we wanted to host it using Wordpress someone would need to configure and manage it.
